src/Http/Routing/src/ArrayBuilder.cs

```diff
@@ -1,6 +1,5 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
```

Comment:
This is incorrect, since it's copied source from something that was MIT.

Reply:
@Pilchie Is there a list of solution filters that I shouldn't touch their license headers?